nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_kernel.py

```diff
@@ -464,7 +464,7 @@ def build(self, target_model: NNCFNetwork) -> ElasticKernelHandler:
             self._node_names_to_make_elastic = [node.node_name for node in all_elastic_kernel_nodes]
 
         for node_name in self._node_names_to_make_elastic:
-            nncf_logger.info("Adding Elastic Kernel op for Conv2D in scope: {}".format(node_name))
+            nncf_logger.info(f"Adding Elastic Kernel op for Conv2D in scope: {node_name}")
```

Comment:
Propose moving to debug
@jpablomch

Reply:
@ljaljushkin Moving to debug works. Thanks!

Reply:
Done